mj_envs/envs/env_base.py

```diff
@@ -98,6 +102,8 @@ def _setup(self,
 
         self.mujoco_render_frames = False
         self.device_id = device_id
+        MjRenderContext(self.sim, device_id=self.device_id)
```

Comment:
@vmoens -- is this something desirable or a debug code?

Reply:
Definitely debug! Let me take it out